datahub-web-react/src/app/entity/shared/EntityDropdown/CreateGlossaryEntityModal.tsx

```diff
@@ -33,6 +35,7 @@ interface Props {
     entityType: EntityType;
     onClose: () => void;
     refetchData?: () => void;
+    clone?: boolean;
```

Comment:
how about a rename to `isCloning`? a little more obvious that it's a boolean flag than just `clone` if you ask me